test/caesium/binding_test.clj

```diff
@@ -80,12 +80,20 @@
              LongLongByReference #{})
            annotations))))
 
+(defn int-func?
+  "checks if given fn is supposed to have an int return type"
+  [f]
+  (let [fns  #{"sodium_init" "crypto_pwhash_argon2i_alg_argon2i13"
+               "crypto_pwhash_alg_default" "crypto_pwhash_alg_argon2i13"
+               "crypto_pwhash_alg_argon2id13" "crypto_pwhash_argon2id_alg_argon2id13"}]
+    (contains? fns f)))
```

Comment:
Please just `(def int-func-name? #{...})` -- use the set directly, not `contains?`